docs/documentation/server_development/topics/identity-brokering/account-linking.adoc

```diff
@@ -16,6 +17,27 @@ There are some preconditions that must be met by the client application before i
 * The user account must already be logged in as an existing user via the OIDC protocol
 * The user must have an `account.manage-account` or `account.manage-account-links` role mapping.
 * The application must be granted the scope for those roles within its access token
+
+The protocol is realized by the link:{adminguide_link}#con-aia_server_administration_guide[Application-initiated action (AIA)]. If you want the user, which is authenticated in your client application, to link
+to the identity provider, it is needed to attach the parameter `kc_action` with the value `idp_link:<identity-provider-alias>` to the OIDC authentication URL and redirect user to this URL. So for example
+to request linking to the identity provider with the alias `my-oidc-provider`, it is needed to attach the parameter like:
+
+[source,subs="attributes+"]
+----
+kc_action=idp_link:my-oidc-provider
+----
+
+==== Refreshing external tokens
+
+If you are using the external token generated by logging into the provider (i.e. a Facebook or GitHub token), you can refresh this token by re-initiating the account linking API.
+
+==== Legacy client initiated account linking
+
+WARNING: The legacy client initiated account linking is using custom protocol, which is not based on AIA. If you are using this protocol, it is recommended to migrate
+your client application to the AIA based protocol described above as legacy client initiated account linking might be removed in the future {project_name} versions.
```

Comment:
```suggestion
your client application to the AIA based protocol described above because legacy client initiated account linking might be removed in the future {project_name} versions.
```